web/controllers/skill_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeCorps.SkillController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Skill
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create]
+
+  def index(conn, params) do
+    skills =
+      case params do
+        %{"filter" => filter} ->
+          ids = String.split(filter, ",") |> Enum.map(fn(n) -> String.to_integer(n) end)
```

Comment:
I would maybe ask something on Stack Overflow to see if someone else has a consistently good way of doing this. This seems like a wheel that probably _shouldn't_ need reinvented given that there are plenty of people using Phoenix with coalesced ids from Ember.